src/jit/gentree.h

```diff
@@ -1459,9 +1454,10 @@ struct GenTree
             case GT_INTRINSIC:
             case GT_LEA:
             case GT_STOREIND:
-            case GT_INITBLK:
-            case GT_COPYBLK:
-            case GT_COPYOBJ:
+            case GT_BLK:
+            case GT_OBJ:
```

Comment:
Hmm. I don't think the the non-store variants belong  belong here. I'll remove them.

---
In reply to: [77282926](https://github.com/dotnet/coreclr/pull/7027#discussion_r77282926) [](ancestors = 77282926)